code/processes/tailer.q

```diff
@@ -0,0 +1,55 @@
+\d .wdb
```

Comment:
Do we not want these functions under a tailer namespace?
```suggestion
\d .tailer
```

Reply:
as some some of the functions are updating functions present in the wdb namespace, I placed the script in that namespace

Reply:
So would you not then want the tailer specific ones in a tailer namespace rather than wdb?